catroid/src/main/java/org/catrobat/catroid/ui/adapter/LookAdapter.java

```diff
@@ -58,15 +58,7 @@ public long getItemId(int position) {
 		if (position < 0 || position >= idMap.size()) {
 			return INVALID_ID;
 		}
-
 		LookData item = getItem(position);
-		if (!idMap.containsKey(item)) {
```

Comment:
apparently removing this causes a crash when editing a look in PocketPaint, renamin still works with it.
i know this is ugly code, but is there a specific reason this was removed?